### PR TITLE
Corrections to Chapter Userguide/digitizing

### PR DESCRIPTION
**Pull request by urskaufmann**
Corrections and additions

### CODE REVIEW

**Comment by ponceta** [comment]
This can also be tested with the html artifact :
https://github.com/teksi/wastewater/actions/runs/9834002894/artifacts/1676269762

File: plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py
Comment (sjib) [inline plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py:1]: this file should not be changed in this doc pull request

File: plugin/teksi_wastewater/interlis/interlis_importer_exporter.py [hunk 1]
```diff
@@ -183,8 +183,6 @@ def interlis_export(
             labels_file_path = os.path.join(tempdir.name, "labels.geojson")
             self._export_labels_file(
                 limit_to_selection=limit_to_selection,
-                # neu export orientation
```
Comment (sjib) [inline plugin/teksi_wastewater/interlis/interlis_importer_exporter.py:186]: this file should not be changed in this doc pull request

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ repos:
 
   # Fix end of files
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+    rev: v4.6.0
```
Comment (sjib) [inline .pre-commit-config.yaml:11]: this file should not be changed in this doc pull request

File: plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py
Comment (urskaufmann) [reply plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py:1]: I can not imagine, why these "interlis" files are in this pull request. I have just changed files in the doc's. ???
What do I have to do now?

**Comment by urskaufmann** [comment]
should resolve #279

File: plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py
Comment (ponceta) [reply plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py:1]: This is only due to the fact that this branch was not up to date with MAIN code, no worry about that, it just require some rebase (merge of main into the development branch) before the final merge. As I just did ;)

**Review by ponceta — APPROVED** [review]

**Pull request merged**